mobile/lib/main.dart

```diff
@@ -87,149 +88,142 @@ class MainApp extends HookWidget {
-                        Expanded(
-                          child: Padding(
-                            padding: const EdgeInsets.symmetric(horizontal: 8),
-                            child: ColorShadowedCard(
-                              shadowColor: context.colors.highlightBlue,
-                              footer: Padding(
-                                padding:
-                                    const EdgeInsets.symmetric(vertical: 2),
-                                child: Text(
-                                  'Stopka',
-                                  style: context.textStyles.textSmall,
-                                ),
+                      ),
+                      Expanded(
+                        child: Padding(
+                          padding: const EdgeInsets.symmetric(horizontal: 8),
+                          child: ColorShadowedCard(
+                            shadowColor: context.colors.highlightBlue,
+                            footer: Padding(
+                              padding: const EdgeInsets.symmetric(vertical: 2),
+                              child: Text(
+                                'Stopka',
+                                style: context.textStyles.textSmall,
                               ),
-                              child: Container(
-                                padding: const EdgeInsets.all(20),
-                                child: Text(
-                                  'Lorem ipsum',
-                                  style: context.textStyles.textSmall,
-                                ),
+                            ),
+                            child: Container(
+                              padding: const EdgeInsets.all(20),
+                              child: Text(
+                                'Lorem ipsum',
+                                style: context.textStyles.textSmall,
                               ),
                             ),
                           ),
                         ),
-                      ],
-                    ),
+                      ),
+                    ],
                   ),
-                  Padding(
-                    padding:
-                        const EdgeInsets.symmetric(vertical: 8, horizontal: 16),
-                    child: ColorShadowedCard(
-                      shadowColor: context.colors.highlightRed,
-                      child: Container(
-                        padding: const EdgeInsets.all(20),
-                        child: Text(
-                          'hello world',
-                          style: context.textStyles.textSmall,
-                        ),
+                ),
+                Padding(
+                  padding:
+                      const EdgeInsets.symmetric(vertical: 8, horizontal: 16),
+                  child: ColorShadowedCard(
+                    shadowColor: context.colors.highlightRed,
+                    child: Container(
```

Comment:
4
same size container with slightly different text appears 4 times, consider separate class with default values of 'lorem ipsum' and padding